src/binder/bind/bind_import_database.cpp

```diff
@@ -0,0 +1,39 @@
+#include <fstream>
+#include <iostream>
+
+#include "binder/copy/bound_import_database.h"
+#include "binder/query/bound_regular_query.h"
+#include "common/exception/binder.h"
+#include "parser/parser.h"
+#include "parser/port_db.h"
+
+using namespace kuzu::binder;
+using namespace kuzu::common;
+using namespace kuzu::parser;
+using namespace kuzu::catalog;
+using namespace kuzu::transaction;
+using namespace kuzu::storage;
+
+namespace kuzu {
+namespace binder {
+
+std::unique_ptr<BoundStatement> Binder::bindImportDatabaseClause(const Statement& statement) {
+    auto& importDatabaseStatement = ku_dynamic_cast<const Statement&, const ImportDB&>(statement);
+    auto boundFilePath = importDatabaseStatement.getFilePath();
+    std::vector<std::string> files = {"schema.cypher", "copy.cypher", "macro.cypher"};
+    std::string finalQueryStatements;
+    for (auto& file : files) {
+        auto filePath = vfs->joinPath(boundFilePath, file);
+        std::ifstream cypherFile(filePath);
```

Comment:
Let's open with our file system. And I think it should handle the case if a file cannot be opened.